lib/rails_admin/config/fields/types/text.rb

```diff
@@ -26,6 +26,12 @@ class Text < RailsAdmin::Config::Fields::Base
             end
           end
 
+          [:froala, :froala_config_options, :froala_css_location, :froala_js_location].each do |key|
+            register_deprecated_instance_option key do
+              fail("The 'field(:foo){ froala true }' style DSL is deprecated. Please use 'field :foo, :froala' instead.")
+            end
+          end
```

Comment:
This part is compatibility purpose for older RailsAdmin configuration(there was [a DSL change](https://github.com/sferik/rails_admin/commit/967efc6f875c5f2075c697e22e894cad2fbc99b1) in 0.5.0), so not required for a newly introduced feature.